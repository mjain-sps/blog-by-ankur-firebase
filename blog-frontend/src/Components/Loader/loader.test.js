import React from "react";
import { render } from "@testing-library/react";
import "@testing-library/jest-dom/extend-expect";
import LoaderComponent from "./loader.component";

test("check snapshot of loader component", () => {
  const { container } = render(<LoaderComponent />);
  expect(container).toMatchSnapshot();
});
