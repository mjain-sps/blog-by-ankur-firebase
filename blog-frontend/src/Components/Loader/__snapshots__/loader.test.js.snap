// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`check snapshot of loader component 1`] = `
<div>
  <div
    class="sc-bdnxRM igcGgs"
  >
    <div
      class="sc-gtsrHT gHGYp"
    >
      <div
        class="sc-dlnjwi kWgJn"
      />
      <div
        class="sc-dlnjwi kWgJn"
      />
      <div
        class="sc-dlnjwi kWgJn"
      />
    </div>
  </div>
</div>
`;
